main.js

```diff
@@ -0,0 +1,28 @@
+function verifyEmail() {
+  var status= document.getElementById('emailStatus');
+
+  var emailPattern=/^([a-zA-Z0-9_.-])+@([a-zA-Z0-9_.-])+\.([a-zA-Z])+([a-zA-Z])+/;
+  if (emailPattern.test(subscribeField.value)) {
+    status.classList.remove("notApproved");
+    status.classList.add("approved");
+  } else {
+    status.classList.remove("approved");
+    status.classList.add("notApproved");
+  }
+}
+var elEmail = document.getElementById('subscribeField')
+elEmail.addEventListener('keyup', verifyEmail, false);
+
+function submitMessage (e) {
```

Comment:
good job using keyup to check immediately instead of just on submit. I feel that's a much better user experience.